docs/developer/plugin-list.asciidoc

```diff
@@ -40,6 +40,10 @@ as uiSettings within the code.
 |Console provides the user with tools for storing and executing requests against Elasticsearch.
 
 
+|{kib-repo}blob/{branch}/src/plugins/controls/readme.mdx[controls]
+|undefined
```

Comment:
did this get generated properly if it's missing a description?

Reply:
Oh true! Let me write out a little MDX readme - I think the problem here is that there is a readme with no content.